website/docs/02 - Connection.md

```diff
@@ -27,14 +27,15 @@ Connection.createConnection(mvisUri: string, account: string, options?: CreateCo
+| `cacheMaxAge`          | `number`      | `3600`                  | The maximum age of cached connection information, such as the current database date                                                                                                                                                                                                                                                                                                                                                    |
+| `timeout`              | `number`      | `0`                     | The request timeout in milliseconds (0 to disable)                                                                                                                                                                                                                                                                                                                                                                                     |
+| `httpAgent`            | `http.Agent`  |                         | An `http.Agent` instance to use with http requests (recommended)                                                                                                                                                                                                                                                                                                                                                                       |
+| `httpsAgent`           | `https.Agent` |                         | An `https.Agent` instance to use with https requests (recommended)                                                                                                                                                                                                                                                                                                                                                                     |
+| `maxReturnPayloadSize` | `number`      | `100_000_000`           | The maximum allowed return payload size in bytes. If this size is exceeded a DbServerError will be thrown. Returning large payloads can have a significant impact on performance and is often the result of invalid database records or an improperly configured query; ie forgetting to use pagination. Tune this value to match your datasets and the type of queries you issue. This can also be configured on a per request basis. |
+| `requestId`            | `string`      | randomly generated UUID | A request/trace ID to be passed to MVIS as a request header with the key `X-MVIS-Trace-Id`                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
This seems to be documented in the wrong place.  For `Connection` it looks like `open` was modified to accept options, but `createConnection` wasn't changed.

Reply:
I also missed the `getDbServerInfo` method documentation updates. Connection.md should be good now.